src/balances/utils.ts

```diff
@@ -133,7 +133,10 @@ const executeMulticall = async (
   // Collect calls we want to make
   const calls: Array<Call> = []
   tokens.map((token) => {
-    if (token.address === constants.AddressZero) {
+    if (
+      token.address === constants.AddressZero ||
+      token.address == '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee'
```

Comment:
Can we move this to constants as well?

Reply:
Changed it: Added a function for zero address checks

Reply:
Great!